ui/components/component-library/icon/icon.types.ts

```diff
@@ -0,0 +1,178 @@
+  ArrowUp = 'arrow-up',
+  BankToken = 'bank-token',
+  Bank = 'bank',
+  Book = 'book',
+  Bookmark = 'bookmark',
+  Bridge = 'bridge',
+  Calculator = 'calculator',
+  CardPos = 'card-pos',
+  CardToken = 'card-token',
+  Card = 'card',
+  Category = 'category',
+  Chart = 'chart',
+  Check = 'check',
+  Clock = 'clock',
+  Close = 'close',
+  CodeCircle = 'code-circle',
+  Coin = 'coin',
+  Confirmation = 'confirmation',
+  Connect = 'connect',
+  CopySuccess = 'copy-success',
+  Copy = 'copy',
+  Danger = 'danger',
+  Dark = 'dark',
+  Data = 'data',
+  Diagram = 'diagram',
+  DocumentCode = 'document-code',
+  Edit = 'edit',
+  Eraser = 'eraser',
+  Ethereum = 'ethereum',
+  Expand = 'expand',
+  Explore = 'explore',
+  Export = 'export',
+  EyeSlash = 'eye-slash',
+  Eye = 'eye',
+  Filter = 'filter',
+  Flag = 'flag',
+  FlashSlash = 'flash-slash',
+  Flash = 'flash',
+  FullCircle = 'full-circle',
+  Gas = 'gas',
+  GlobalSearch = 'global-search',
+  Global = 'global',
+  Graph = 'graph',
+  Hardware = 'hardware',
+  Heart = 'heart',
+  Hierarchy = 'hierarchy',
+  Home = 'home',
+  Import = 'import',
+  Info = 'info',
+  Key = 'key',
+  Light = 'light',
+  Link = 'link',
+  Loading = 'loading',
+  LockCircle = 'lock-circle',
+  LockSlash = 'lock-slash',
+  Lock = 'lock',
+  Login = 'login',
+  Logout = 'logout',
+  Menu = 'menu',
+  MessageQuestion = 'message-question',
+  Messages = 'messages',
+  MinusSquare = 'minus-square',
+  Minus = 'minus',
+  Mobile = 'mobile',
+  Money = 'money',
+  Monitor = 'monitor',
+  MoreHorizontal = 'more-horizontal',
+  MoreVertical = 'more-vertical',
+  NotificationCircle = 'notification-circle',
+  Notification = 'notification',
+  PasswordCheck = 'password-check',
+  People = 'people',
+  ProgrammingArrows = 'programming-arrows',
+  Question = 'question',
+  Received = 'received',
+  Refresh = 'refresh',
+  Save = 'save',
+  ScanBarcode = 'scan-barcode',
+  ScanFocus = 'scan-focus',
+  Scan = 'scan',
+  Scroll = 'scroll',
+  Search = 'search',
+  SecurityCard = 'security-card',
+  SecurityCross = 'security-cross',
+  SecurityKey = 'security-key',
+  SecuritySearch = 'security-search',
+  SecuritySlash = 'security-slash',
+  SecurityTick = 'security-tick',
+  SecurityTime = 'security-time',
+  SecurityUser = 'security-user',
+  Security = 'security',
+  Send1 = 'send-1',
+  Send2 = 'send-2',
+  Setting = 'setting',
+  Slash = 'slash',
+  SnapsMobile = 'snaps-mobile',
+  SnapsPlus = 'snaps-plus',
+  Snaps = 'snaps',
+  Speedometer = 'speedometer',
+  Star = 'star',
+  Student = 'student',
+  SwapHorizontal = 'swap-horizontal',
+  SwapVertical = 'swap-vertical',
+  Tag = 'tag',
+  Tilde = 'tilde',
+  Timer = 'timer',
+  Trash = 'trash',
+  TrendDown = 'trend-down',
+  TrendUp = 'trend-up',
+  UserCircleAdd = 'user-circle-add',
+  UserCircle = 'user-circle',
+  User = 'user',
+  WalletCard = 'wallet-card',
+  WalletMoney = 'wallet-money',
+  Wallet = 'wallet',
+  Warning = 'warning',
+}
```

Comment:
Updating IconNames to manual process. This decision was made as there are not a lot of new icons going in. This may become a problem if we do some batch updates but right now I think it is ok.